### DISCUSSION TITLE
"Edge Requests" Graph y-axis is nonsensical

### DISCUSSION THREAD

**Discussion by danielbeardsley** [vercel/community#7247] [Bug Report]
### Page to Investigate

https://vercel.com/ifixit/~/usage#networking-edge-requests

### Steps to Reproduce

Look at this graph and try to make sense of the Y-axis:

* It's not linear...
* It's not logarithmic...
* It's not sensical!

![image](https://github.com/vercel/community/assets/26855/eafbaab1-cf5a-4267-b50f-37d35f7196b3)

I assume this is because of some very poorly done rounding on the Y-axis and not some nefarious ploy to hide information.

I think in actuality it's linear and the `1M` is a poorly rounded `1200k` and `2M` is a poorly rounded `1500k`.

**Comment by amyegan** [comment]
Thanks for reporting this! I've created a ticket and we're looking into it

[a comment by an automated account is not shown]
